core/nodepool/cluster/cluster_test.go

```diff
@@ -106,6 +106,7 @@ func TestValidateKeyPair(t *testing.T) {
 s3URI: s3://mybucket/mydir
 apiEndpoints:
 - name: public
+  nodePoolRollingStrategy: parallel
```

Comment:
Nit, but this seems redundant because it is a config for an API endpoint, that does nothing to do with the feature